pkg/browsers/windows.go

```diff
@@ -0,0 +1,35 @@
+package browsers
```

Comment:
Nitpick but should we put these structs into the osx file rather than the windows onw

Reply:
Good pickup, I hadn't checked where these belonged, assumed it was windows